recipes/perl-graph-easy/0.76/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "perl-graph-easy" %}
```

Comment:
Could you submit perl-graph-easy to conda-forge instead? We're trying to slowly move all of the non-bioinfo perl packages over there.

Reply:
Ah, yes, I was wondering if that might be the case based on the guidelines, but wasn't sure since a lot of perl packages are here. I've taken it out of this PR and will submit it there instead.